doc/test-framework/scorecard.md

```diff
@@ -79,29 +84,245 @@ API server, indicating that it is modifying resources. This test has a maximum s
+This test checks that the CSV has an [`alm-examples` section][alm-examples] for each CR provided by the `--cr-manifest` flag in its metadatas' annotations. This test has a maximum score
+equal to the number of CRs provided via the `--cr-manifest` flag.
 
 #### Spec Fields With Descriptors
 
-This test verifies that every field in the Custom Resource's spec section has a corresponding descriptor listed in
-the CSV. This test has a maximum score equal to the number of fields in the spec section of your Custom Resource.
+This test verifies that every field in the Custom Resources' spec sections have a corresponding descriptor listed in
+the CSV. This test has a maximum score equal to the total number of fields in the spec sections of each custom resource passed in via the `--cr-manifest` flag.
 
 #### Status Fields With Descriptors
 
-This test verifies that every field in the Custom Resource's status section has a corresponding descriptor listed in
-the CSV. This test has a maximum score equal to the number of fields in the status section of your Custom Resource.
+This test verifies that every field in the Custom Resources' status sections have a corresponding descriptor listed in
+the CSV. This test has a maximum score equal to the total number of fields in the status sections of each custom resource passed in via the `--cr-manifest` flag.
+
+## Extending the Scorecard with Plugins
+
+To allow the scorecard to be further extended and capable of more complex testing as well as allow the community to make their own scorecard tests, a plugin system has been implemented
+for the scorecard. To use it, a user simply needs to add a binary or script to a `scorecard/bin` directory in their operator's root directory that runs the plugin. It is
+recommended to run plugins with a script to allow more configuration from the user and place other required assets in another subdirectory in the `scorecard` directory.
+For instance, `scorecard/assets`. The scorecard runs all plugins from the root `scorecard` directory. Since the scorecard will run all executable files in the scorecard's
+`bin` subdirectory, the plugins can be written in any programming language supported by the OS the scorecard is being run on.
+
+To provide results to the scorecard, the plugin must output a valid JSON object to its `stdout`. Invalid JSON in the `stdout` will result in the plugin being marked as failed.
+To provide logs to the scorecard, plugins can either set the `log` field for the scorecard suites they return or they can output logs to `stderr`, which will stream the log
+to the console if the scorecard is being run in with `human-readable` output or be added to the main ScorecardOutput `log` field when being run with the output flag set to `json`.
+
+### JSON format
+
+The JSON output is formatted in the same way that a Kubernetes API would be, which allows us to make updates and new versions of the schema if necessary as well as use various
```

Comment:
Trying to avoid the use of "us".
```suggestion
The JSON output is formatted in the same way that a Kubernetes API would be, which allows for updates to the schema as well as the use of various
```